recipes-extended/xen/xen-openxt.inc

```diff
@@ -11,7 +11,7 @@ EXTRA_OECONF_remove = " \
     --disable-pv-grub \
     --disable-xenstore-stubdom \
     --disable-ocamltools \
-    --with-initddir=${INIT_D_DIR} \
+    --with-initddir=${sysconfdir}/init.d \
```

Comment:
Why? This is the value that is set by the update-rc.d bbclass. Is it getting trampled elsewhere?

Reply:
This was in response to feedback from @eric-ch on the last round, where he'd had to chase down where the definition for `INIT_D_DIR` was set and what it was. I don't actually care either way, so I'll drop this commit that touches this, and leave it as `INIT_D_DIR`.

Reply:
> That looks fine though, INIT_D_DIR is defined in updated-rc.d.bbclass to ${sysconfdir}/init.d.

I don't quite remember what I meant... Maybe because this `.inc` does not inherit `update-rc.d`?

Reply:
oh... you're right, @eric-ch ! nice catch, thanks. I need to look at this.

Reply:
Then IMHO if we are touching/manipulating init in the `.inc` then this should inherit update-rc.d.

Reply:
Patch just submitted to meta-virtualization here to clean this up. Moves the tools-specific configuration for the init script handling over into the tools recipe and out of the common include, so the hypervisor recipe doesn't need that variable defined:
https://lists.yoctoproject.org/g/meta-virtualization/message/5663

Reply:
Looking into this now: the dunfell branch of meta-virtualization doesn't include my change above, so `xen.inc` there contains:
`    --with-initddir=${INIT_D_DIR} \
`
and `INIT_D_DIR` is not defined when `update-rc.d` is not included: in dunfell meta-virt, the xen-tools recipe has it set, but the xen (hypervisor) recipe does not. This `xen-open.inc` file is only included by the hypervisor recipe in OpenXT, not any of the others.

I'm going to drop this change since the existing logic does the right thing for setting the `EXTRA_OECONF` variable: it removes the `--with-initddir` setting for the Xen hypervisor recipe.

I've also just found that the `xen-ocaml-libs` recipe needs to inherit `update-rc.d` to set `INIT_D_DIR` correctly.

_(Note that the Zeus branch of meta-virtualization does not contain the split xen / xen-tools recipes, but the Dunfell branch of meta-virtualization builds Xen OK with the Zeus branch of the other upstream repos; hence looking at Dunfell as the stable target for meta-virtualization for this PR.)_